orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -548,8 +553,8 @@ def export_data(self):
 
 PREPROCESSOR_MODULES = [
     TokenizerModule,
-    TransformationModule,
     CasingModule,
+    TransformationModule,
```

Comment:
Nevermind, now I know why changing the order actually reintroduced the bug after rebasing to master hah. Will discard this change.